cpg-core/src/main/java/de/fraunhofer/aisec/cpg/frontends/cpp/DeclaratorHandler.kt

```diff
@@ -166,13 +168,16 @@ class DeclaratorHandler(lang: CXXLanguageFrontend) :
         lang: LanguageFrontend,
         ctx: IASTNode,
     ): MethodDeclaration {
-        // check, if its a constructor
-        return if (name == recordDeclaration?.name) {
-            NodeBuilder.newConstructorDeclaration(name, null, recordDeclaration, lang, ctx)
-        } else NodeBuilder.newMethodDeclaration(name, null, false, recordDeclaration, lang, ctx)
+        // Check, if it's a constructor
+        val method =
+            if (name == recordDeclaration?.name) {
+                NodeBuilder.newConstructorDeclaration(name, null, recordDeclaration, lang, ctx)
```

Comment:
Remove `NodeBuilder` consistently (as in line 105) and use `{}` for the else branch too?

Reply:
Done